src/common/components/paymentMethods/creditCardForm/creditCardForm.tpl.html

```diff
@@ -14,16 +14,16 @@ <h4 class="panel-title visible" translate>{{'CREDIT_CARD_PAYMENT'}}</h4>
       <div class="col-sm-6">
         <div class="form-group is-required" ng-class="{'has-error': ($ctrl.creditCardPaymentForm.cardNumber | showErrors)}">
           <label>
-            <span translate>{{'CARD_NUM'}}</span>
+            <span translate>{{'CARD_NUM_FULL'}}</span>
             <input type="text"
-                   name="cardNumber"
-                   autocomplete="cc-number"
-                   class="form-control  form-control-subtle"
-                   ng-model="$ctrl.creditCardPayment.cardNumber"
-                   ng-attr-placeholder="************{{$ctrl.creditCardPayment.cardNumberPlaceholder}}"
-                   ng-required="!$ctrl.paymentMethod || $ctrl.creditCardPayment.cardNumber"
-                   ng-disabled="$ctrl.disableCardNumber"
-                   credit-card-number>
+                  name="cardNumber"
+                  autocomplete="cc-number"
+                  class="form-control  form-control-subtle"
+                  ng-model="$ctrl.creditCardPayment.cardNumber"
+                  ng-attr-placeholder="************{{$ctrl.creditCardPayment.cardNumberPlaceholder}}"
+                  ng-required="!$ctrl.paymentMethod || $ctrl.creditCardPayment.cardNumber"
+                  ng-disabled="$ctrl.disableCardNumber"
+                  credit-card-number>
```

Comment:
Looks like we lost the alignment here.

Reply:
If it was me, I would do a rebase with an `edit` to the first commit, fixing the HTML structure there, then modify the fix structure commit to remove this change from it. When modifying the first commit, the changes could be rolled back and then just change the translation reference. Something to the effect of:

```sh
git rebase -i head~8
[[ indicate desire to modify first commit ]]
git reset head^
git checkout src/common/components/paymentMethods/creditCardForm/creditCardForm.tpl.html
[[ change translation reference ]]
git add -A
git commit -m 'Updated # to Number'
git rebase --continue
```

At this point, there would be conflicts again to fix up, but I believe you can just "accept current changes" for each time.

Reply:
Yes indeed!